<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaInfoDepartamento</class>
 <widget class="QDialog" name="VentanaInfoDepartamento">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>268</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>475</width>
    <height>268</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>475</width>
    <height>268</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Informaci√≥n del departamento</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../recursos/qtRecursosIconos.qrc">
    <normaloff>:/iconos/iconos/icono-app-dgtic.png</normaloff>:/iconos/iconos/icono-app-dgtic.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#VentanaInfoDepartamento {
	background-color: #fff;
}

#frameGeneralInfoDepartamento .QPushButton{
	color: #ffffff;
	background-color: #417fd2;
	border: 1px solid rgb(0, 0, 0);
	border-radius: 6px;
	padding-bottom: 2px;
	font: 8.5pt &quot;Segoe UI Black&quot;;
}

#frameGeneralInfoDepartamento .QPushButton::hover{
	background-color: #5c8bc9;
}

#frameGeneralInfoDepartamento .QPushButton::pressed{
	background-color: #063c85;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frameGeneralInfoDepartamento">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,1">
      <item>
       <widget class="QFrame" name="frameInfoDepartamento">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>70</y>
           <width>371</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="hlInfoDepartamento">
          <item>
           <widget class="QLabel" name="labelInfoNombreDepartamento">
            <property name="text">
             <string>Nombre del departamento:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputInfoNombreDepartamento"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frameAccionesInfoDepartamento">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>20</y>
           <width>371</width>
           <height>41</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="hlBotonesAccionDepartamento">
          <item>
           <widget class="QPushButton" name="botonAcualizarInfoDepartamento">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Actualizar info</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="botonEliminarDepartamento">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Eliminar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="botonCancelarInfoDepartamento">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Cancelar</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../recursos/qtRecursosIconos.qrc"/>
 </resources>
 <connections/>
</ui>
